src/Symfony/Component/Security/Http/AccessToken/Oidc/OidcUserInfoTokenHandler.php

```diff
@@ -49,7 +49,7 @@ public function getUserBadgeFrom(string $accessToken): UserBadge
 
             // UserLoader argument can be overridden by a UserProvider on AccessTokenAuthenticator::authenticate
             return new UserBadge($claims[$this->claim], fn () => $this->createUser($claims), $claims);
-        } catch (\Throwable $e) {
+        } catch (\Exception $e) {
```

Comment:
we should never catch `Throwable` unless reasons which should be explicit/obvious